lib/transforms/drawGraph.js

```diff
@@ -139,19 +138,19 @@ module.exports = targetFileName => {
     ]);
     dotProcess.stdin.end(dotSrc);
     dotProcess.stderr.on('data', chunk => console.warn(`dot STDERR: ${chunk}`));
-    await Promise.fromNode(cb => {
+    await new Promise(resolve => {
       dotProcess.on('error', error => {
         if (error.code === 'ENOENT') {
-          cb(new Error('"dot" not found. Please install graphviz.'));
+          throw new Error('"dot" not found. Please install graphviz.');
```

Comment:
These `throw`s will turn into unhandled exceptions. They should reject the promise instead.

Reply:
Fixed in dcd95269